opennms-base-assembly/src/main/filtered/etc/thresholds.xml

```diff
@@ -34,7 +34,7 @@
       </threshold>
       <threshold description="Trigger an alert when the percentage of disk space used on any disk increases by a relative 33.3% compared to its most recent previous measurement (that is, there is suddenly less free space)" type="relativeChange" ds-type="dskIndex" value="1.333" rearm="0.0" trigger="1" ds-label="ns-dskPath" filterOperator="OR" ds-name="ns-dskPercent"/>
       <threshold description="Trigger an alert when the percentage of inodes used on any disk increases by a relative 33.3% compared to its most recent previous measurement (that is, there are suddenly fewer free inodes)" type="relativeChange" ds-type="dskIndex" value="1.333" rearm="0.0" trigger="1" ds-label="ns-dskPath" filterOperator="OR" ds-name="ns-dskPercentNode"/>
-      <expression description="Trigger an alert when the five minute CPU load average metric reaches or goes above 70% for two consecutive measurement intervals" type="high" ds-type="node" value="70.0" rearm="50.0" trigger="2" filterOperator="OR" expression="(loadavg5 / CpuNumCpus ) * 100.0"/>
+      <expression description="Trigger an alert when the five minute CPU load average metric reaches or goes above 70% for two consecutive measurement intervals" type="high" ds-type="node" value="70.0" rearm="50.0" trigger="2" filterOperator="OR" expression="(loadavg5 / 100) / CpuNumCpus )) * 100.0"/>
```

Comment:
the number of brackets in the expression looks strange to me :) 1 open and 3 closings :)

`(loadavg5 / 100) / CpuNumCpus )) * 100.0`